kroxylicious/src/main/java/io/kroxylicious/proxy/internal/codec/ByteBufAccessorImpl.java

```diff
@@ -22,10 +22,11 @@
  * depends on NIO ByteBuffer, so copying between ByteBuffer and ByteBuf cannot
  * always be avoided.
  */
-public class ByteBufAccessor implements Readable, Writable {
+public class ByteBufAccessorImpl implements ByteBufAccessor, Readable {
```

Comment:
Why do we have an asymmetry here between the Readable and Writable now?

Reply:
Because clients of `ByteBufAccessor` currently only access the `Writable` methods, so I limited what it exposes (following my general preference for being restrictive in terms of what gets exposed). Do you see a need for exposing `Readable` there as well?